lib/pure/parsecsv.nim

```diff
@@ -29,8 +32,8 @@
 ## reference for item access with `rowEntry <#rowEntry.CsvParser.string>`_:
```

Comment:
Links should have comma (`,`) not dot (`.`):

```
`rowEntry <#rowEntry,CsvParser,string>`_
```

This should be fixed throughout the document.